libs/application/templates/passport-annulment/src/lib/PassportAnnulmentTemplate.ts

```diff
@@ -84,9 +83,9 @@ const PassportAnnulmentTemplate: ApplicationTemplate<
               label: m.actionCardDoneTag,
             },
           },
-          onEntry: defineTemplateApi({
+          /*onEntry: defineTemplateApi({
             action: ApiActions.submitApplication,
-          }),
+          }),*/
```

Comment:
Is this onEntry supposed to be disabled?
Is this temporary or should it be removed?

Reply:
temp, will be addressed in next PR